EsperBenchmark/FilterBenchmark/FilterBenchmark.iml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the iml files. These are from IDEA which are not required to be committed here.